### PR TITLE
Profile revised

### PR DESCRIPTION
**Pull request by dallasjc**
Adds icons with tooltips tracking various metrics related to each user (votes, comments, proposals, proxies)

Removes the "This is your profile" box from the top and redistributes the info to other places
*gravatar info goes on edit profile page
*added social links with descriptor about public votes
*added edit profile button

### CODE REVIEW

**Comment by dallasjc** [comment]
image

**Comment by dallasjc** [comment]
I'd like to change the icons to outlined, but can't figure out how

**Comment by dsernst** [comment]
FontAwesome provides outline versions of the icons. I don't believe it's for all of them though. Usually you just add `-o` to the end of the icon name to use the outline version.

I would check the icon library website (remember we're using FontAwesome v4) to see what's available.

File: components/UserProfilePage.js [hunk 1]
```diff
@@ -3,18 +3,24 @@ const Component = require('./Component')
 const Comment = require('./Comment')
 
 module.exports = class UserProfilePage extends Component {
-  render() {
-    const { proxied_name, selected_profile: p, user } = this.state
+  oninit() {
+    if (!this.state.user) return this.location.redirect('/sign_in')
```
Comment (alexmingoia) [inline components/UserProfilePage.js]: This shouldn't be here. This redirects to the sign-in page, but user profiles should be visible to anyone.

**Comment by alexmingoia** [comment]
I checked this out to try and merge this, and ran into a few issues:

- User profiles don't load when signed-out, due to some code referencing the user.
- Some of the tooltip text is cutoff for the "Directly representing" and "Indirectly representing" tooltips (using Firefox 66).
- When viewing my own profile page, the tooltip for the "Edit Profile" button is showing up when hovering over the "Proxy to Alex" button.
- `components/SignUpSidebar.js` is not used and should be removed.
- The tooltips should be able to be toggled by tapping so that they work on mobile.

**Pull request merged**